Project02Eclipse/src/com/hemebiotech/analytics/AnalyticsCounterProgram.java

```diff
@@ -0,0 +1,32 @@
+package com.hemebiotech.analytics;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Hashtable;
+
+public class AnalyticsCounterProgram {
+    ISymptomReader reader;
+    ISymptomResultWriter writer;
+    SymptomsSort sorter = new SymptomsSort();
+    ArrayList<String> listSymptoms = new ArrayList<String>();
```

Comment:
Thos 2 properties can be declared directly inside the start method, since they don't need to be injected at this time.